### PR TITLE
install @angular/http

### PR DESCRIPTION
**Pull request by kevinwdev6**
I have to install @angular/http in order for angular-in-memory-web-api to work.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by kevinwdev6** [comment]
I signed it!

File: aio/content/tutorial/toh-pt6.md [hunk 1]
```diff
@@ -50,10 +50,11 @@ If you're _coding along_ with this tutorial, stay here and add the *In-memory We
 
 </div>
 
-Install the *In-memory Web API* package from _npm_
+Install the *In-memory Web API* package and *@angular/http* package from _npm_
 
 <code-example language="sh" class="code-shell">
   npm install angular-in-memory-web-api --save
+  npm install @angular/http
```
Comment (Gillespie59) [inline aio/content/tutorial/toh-pt6.md]: Could you add the @angular/http module into the previous npm command ?
And also, remove the useless --save options ?

Comment (kevinwdev6) [reply aio/content/tutorial/toh-pt6.md]: Updated

**Comment by brandonroberts** [comment]
`@angular/http` is only required with versions of `angular-in-memory-web-api`. The latest version supports both `@angular/http` and `@angular/common/http`. The tutorial uses a newer version.

**Pull request closed without merging**